web/pgadmin/browser/server_groups/servers/databases/publications/__init__.py

```diff
@@ -710,38 +868,97 @@ def get_tables(self, gid, sid, did):
                               server_type=self.manager.server_type
                               )
         status, rset = self.conn.execute_2darray(sql)
+        if not status:
+            return internal_server_error(errormsg=res)
+
+        if self.manager.version < 150000:
+            for row in rset['rows']:
+                res.append(
+                    {
+                        'label': row['table'],
+                        'value': row['table']
+                    }
```

Comment:
you can simply change the old SQL to add `tid` and make the code common instead of adding so many if conditions.